CRM/Activity/Form/Task/PDFLetterCommon.php

```diff
@@ -69,7 +69,7 @@ public static function createTokenProcessor() {
     return new TokenProcessor(\Civi::dispatcher(), [
       'controller' => get_class(),
       'smarty' => FALSE,
-      'schema' => ['activityId'],
+      'schema' => ['activity_id'],
```

Comment:
As above, maybe:
```
    'schema' => (array) CRM_Activity_Tokens::getEntityIDFieldName();
```

Reply:
So in this case I don't know it adds value - it's harder to read & it's still hard-coded in the sense that the class is hard-coded